### DISCUSSION TITLE
What's a good way to include a "home" link in Reveal.js slides?

### DISCUSSION THREAD

**Discussion by andrewcstewart** [quarto-dev/quarto-cli#6433] [Q&A]
I love the ability to include Reveal.js slides in a Quarto site, but after stepping through a slidedeck it's difficult to get back to the main site.  Just curious how other folks typically handle this in their slides.  Maybe include a link back to main site in a _quarto.yml file somewhere?

**Comment by mcanouil** [comment]
The slides footer seems to be a good place.
You can also add some links pretty much anywhere as long as you know HTML/CSS.

Lastly, you could include the slides in an iframe, thus you don't leave the main site.

**Reply by andrewcstewart** [reply]
Yeah, including a default footer is certainly an option.  The use case here is hosting a growing collection of slide decks on a Quarto site, contributed by multiple authors, so the desire is to streamline as much of the navigation/configuration as possible.

I like the iframe idea too and was actually wondering if the Preview Link feature could be part of a solution here.  I have the slide decks listed in a Listing page.. I suppose I could potentially write a custom listing type that uses the Preview Link feature.

Good ideas.  Thanks!
